chain-test/src/test/scala/org/bitcoins/chain/blockchain/ChainHandlerTest.scala

```diff
@@ -34,10 +35,11 @@ class ChainHandlerTest extends ChainDbUnitTest {
 
   override type FixtureParam = ChainHandler
 
-  override val defaultTag: ChainFixtureTag = ChainFixtureTag.GenisisChainHandler
+  override val defaultTag: ChainFixtureTag =
+    ChainFixtureTag.GenesisChainHandlerWithFilter
 
   override def withFixture(test: OneArgAsyncTest): FutureOutcome =
-    withChainHandler(test)
+    withChainHandlerGenesisFilter(test)
```

Comment:
You use this one but don't use the one without the genesis filters ever. Is that intentional?

Reply:
Hmm... no. I would like to re-write the test cases in `ChainHandler` to use `withChainHandler()` but thought this PR is already big enough, we will continue with having the genesis filters for now.